Classes/Controllers/PBGitHistoryController.m

```diff
@@ -401,6 +401,21 @@ - (IBAction)selectPrevious:(id)sender
 	[searchController selectPreviousResult];
 }
 
+- (IBAction) selectParentCommit:(id)sender
+{
+	NSArray *selectedObjects = commitController.selectedObjects;
+	if (selectedObjects.count != 1) return;
+
+	PBGitCommit *selectedCommit = selectedObjects[0];
+
+	NSArray <GTOID *> *parents = selectedCommit.parents;
+	/* TODO: This is a merge commit. It would be nice to choose the parent with
+	 * the most commits, but for now we will use whatever commit is our first parent.
+	 */
+
```

Comment:
Yeah, I'm not even sure git has a default for parent ordering.

It might be possible to make it smarter by asking for a revwalk for all parents and choosing the one that *isn't* the first revwalk results, but it will only work for 2-headed merges (which arguably are the most common). And I wasn't really keen on adding revwalking here 😏.

I don't really understand your behavior question. The plan was being able to move, preserving the "current" lane, so I think that's a yes ?